Specifications/Frontend/Frontend-codespecs.md

```diff
@@ -0,0 +1,38 @@
+# Frontend Code Specifications
+
+## UI
+We will generate graphs on demand using [matplotlib](https://matplotlib.org/tutorials/introductory/usage.html#sphx-glr-tutorials-introductory-usage-py "matplotlib"). The function is triggered by the generate graph button. This will check which attribute's checkboxes are checked and send a list containing keywords to the attributes in question to the api, which requests the data with corresponding keywords.
+
+By hovering your coursor over the graph, a box containing the data point's data appears at the cursor's location. 
+
+By scrolling with your cursor wheel while hovering over, or by moving your fingers in oposite directions on the graph box, you're able to zoom in or out on the graph, and by dragging with a finger or clicking, holding and dragging around with a coursor over the graph you're able to pan over the graph while zoomed in.
+
+* for grahical specs, see [Website Specs file](https://github.com/vigge93/PA1450-Development-task/blob/Specifications-Frontend/Specifications/Frontend/Untitled%20Diagram%20(3).png " Website Specs file").
```

Comment:
Bra

Reply:
Kanske värt att inkludera vart man väljer de olika attributen med?

Reply:
Vad man väljer attribut med står i första stycket: "This will check which attribute's checkboxes are checked and send [...]"